Remora.Discord.Interactivity/Responders/InteractivityResponder.cs

```diff
@@ -169,6 +169,48 @@ or ComponentType.ChannelSelect
         return await TryExecuteInteractionCommandAsync(context, commandPath, parameters, ct);
     }
 
+    private static Result<IReadOnlyDictionary<string, IReadOnlyList<string>>> BuildParametersFromResolvedData
+    (
+        IMessageComponentData data
+    )
+    {
+        var parameters = new Dictionary<string, IReadOnlyList<string>>();
+
+        if (!data.Resolved.IsDefined(out var resolved))
+        {
+            return parameters;
+        }
+
+        if (resolved.Users.IsDefined(out var users))
```

Comment:
This might add more than the selected users to the input parameters if the resolved data contains additional entities. You should probably filter these based on the values from the selection made.

Reply:
I've implemented this check